Formula/go.rb

```diff
@@ -9,9 +9,9 @@ class Go < Formula
       sha256 "6a33569f9d0d21db31614086cc2a4f0fbc683b41c1c53fb512a1341ce5763ff5"
       version "1.15.8"
     else
```

Comment:
Thanks for your PR, @circa10a.

You need to get rid of this entire `if`-`else` block, as it's no longer needed. The `caveats` block also needs to go. There may also be other changes needed that I haven't spotted yet; please double-check.

Reply:
Thanks for the review @carlocab, updated.